lib/codecs/src/decoding/mod.rs

```diff
@@ -345,6 +361,14 @@ impl DeserializerConfig {
         }
     }
 
+    /// Returns an appropriate default framing config for the given deserializer with message based inputs.
+    pub fn default_message_based_framing(&self) -> FramingConfig {
```

Comment:
Moved from a static method, as now it should use a `ChunkedGelf` decoder if the deserializer is gelf and the transport protocol is message-based